platform-includes/sensitive-data/set-user/native.mdx

```diff
@@ -1,11 +1,7 @@
-The Native SDK maintains all data in a single global scope.
```

Comment:
This line seems to have been from a copy-paste from the `Scopes and Hubs` page's code snippet.